app/tests/test_models.py

```diff
@@ -1,7 +1,39 @@
+from flask import Flask
+from flask.ext.testing import TestCase
 from nose.tools import eq_
 
 from .. import models
 
+db = models.db
+
+class DbTestCase(TestCase):
+    BASE_APP_CONFIG = dict(
+        # TODO: We should use a test postgres db instead, since it's
+        # closer to our production deployment.
```

Comment:
Yeah, the sqlite was just a temporary workaround--though now I'm thinking we might want to keep it in there until #23 is fixed, so that at least our Travis CI integration is reliable?